### PR TITLE
fix(V2): small tweaks and fixes for V2

### PR DESCRIPTION
**Pull request by kaosat-dev**
Small sets of fixes for blocker bugs before further changes :)

### All Submissions:

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?
* [x] Does your submission pass tests?

### CODE REVIEW

File: packages/utils/regl-renderer/src/geometry-utils/entitiesFromSolids.js [hunk 1]
```diff
@@ -56,7 +56,7 @@ const entitiesFromSolids = (params, solids) => {
     const visuals = {
       drawCmd: 'drawMesh',
       show: true,
-      color: [0.8, 0.5, 0.7, 0.1],
+      color: meshColor,//[0.8, 0.5, 0.7, 1.0],
       transparent: geometry.isTransparent, // not sure
```
Comment (z3dev) [inline packages/utils/regl-renderer/src/geometry-utils/entitiesFromSolids.js]: I think this is the correct change. Why leave the comment with the old color?

File: packages/utils/regl-renderer/src/rendering/commands/drawMesh/index.js [hunk 2]
```diff
@@ -71,7 +71,7 @@ const drawMesh = function (regl, params = { extras: {} }) {
 
       func: { src: 'src alpha', dst: 'one minus src alpha' }
     },
-    primitive: (context, props) => props && props.primitive ? props.primitive : 'triangles'
+    primitive: (context, props) => props && props.primitive ? props.primitive : (type === '3d' ? 'triangles' : 'lines')
```
Comment (z3dev) [inline packages/utils/regl-renderer/src/rendering/commands/drawMesh/index.js]: Hmm... I understand the change but wonder if the real correction should be made when primitives is called, I.e making sure that props.primitive is set correctly.

File: packages/web/src/ui/flow/design.js [hunk 3]
```diff
@@ -103,7 +103,7 @@ const reducers = {
  * @returns {Object} the updated state
  */
   setDesignContent: (state, payload) => {
-    console.log('design: set content', state.design, payload)
+    console.log('design: set content', state, state.design, payload)
```
Comment (z3dev) [inline packages/web/src/ui/flow/design.js]: I think console messages should be commented out for the final PR.

File: packages/utils/regl-renderer/src/geometry-utils/entitiesFromSolids.js (hunk 1 above)
Comment (kaosat-dev) [reply packages/utils/regl-renderer/src/geometry-utils/entitiesFromSolids.js]: agreed

File: packages/utils/regl-renderer/src/rendering/commands/drawMesh/index.js (hunk 2 above)
Comment (kaosat-dev) [reply packages/utils/regl-renderer/src/rendering/commands/drawMesh/index.js]: that is a different thing, props.primitive is an override, and this sets the **default** for both 2D & 3D

File: packages/web/src/ui/flow/design.js (hunk 3 above)
Comment (kaosat-dev) [reply packages/web/src/ui/flow/design.js]: agreed, but that should be done at a later stage once we are truely in cleanup mode ;)

**Pull request merged**